.github/workflows/docker-mixnet.yml

```diff
@@ -74,6 +74,9 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v4
 
+      - name: Install socat, for talking to management socket
+        run: sudo apt update && sudo apt install -y socat
```

Comment:
ah okay so this installs socat so that you can shell into the docker image and use it from there? or can you socat from the host?